.gitattributes

```diff
@@ -2,4 +2,6 @@
 *.cs diff
 *.csproj diff
 *.sln diff
-*.resx diff
\ No newline at end of file
+*.resx diff
+
+* text=auto
```

Comment:
Merge in latest from main so this doesn't show up as changed (looks like there's some other stuff showing as changed as well, if you do a merge from main it should fix all those)

Reply:
(fixed it for you)